src/Network/Network/help/New-AzApplicationGatewayFirewallPolicyExclusion.md

```diff
@@ -0,0 +1,107 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/new-azapplicationgatewayfirewallexclusionconfig
```

Comment:
URI is incorrect, there is no "config" at the end of cmdlet name

Reply:
Fixed